github-actions/generate-doc/action.yml

```diff
@@ -0,0 +1,47 @@
+name: 'Generate Readme Doc'
```

Comment:
how does this interact with the existing build_docs call in start_release? will it generate duplicate README commits when you merge to next? or are you planning to just remove the build_docs call in start_release and leave just this one?

Reply:
This is basically subset of start_release. When merge to next, doc will re-generate again. I am just afraid if delete build_docs in start_release, I am not sure if it could have other issues. So, I just leave it and no hurt to commit two times.

Reply:
I'm in favor of cleaning it up so there's only one. leaving both is likely to cause confusion down the line if something goes wrong. I don't think anything depends on the one in start_release but if there is then we should just update the flow afterwards

Reply:
yes, but wouldn't that automatically trigger this new workflow since it involves changing the app JSON?

Reply:
Agree. :) I have removed doc update part in start_release.py.